trade-finance-manager-ui/server/controllers/case/amendments/amendmentFacilityEndDate.get.api-test.js

```diff
@@ -23,6 +23,14 @@ const user = {
   email: 'test@localhost',
 };
 
+const gefFacility = {
+  facilitySnapshot: { dates: { coverEndDate: '20 Oct 2020' }, type: FACILITY_TYPE.CASH },
```

Comment:
can this be a const and maybe in test-mocks

Reply:
no longer using